packages/starlight/translations/pt.json

```diff
@@ -16,5 +17,15 @@
   "page.editLink": "Editar página",
   "page.lastUpdated": "Última atualização:",
   "page.previousLink": "Anterior",
-  "page.nextLink": "Próximo"
+  "page.nextLink": "Próximo",
+  "pagefind.clear_search": "Limpar",
+  "pagefind.load_more": "Carregar mais resultados",
+  "pagefind.search_label": "Pesquisar neste site",
+  "pagefind.filters_label": "Filtros",
+  "pagefind.zero_results": "Sem resultados para [SEARCH_TERM]",
+  "pagefind.many_results": "[COUNT] resultados para [SEARCH_TERM]",
+  "pagefind.one_result": "[COUNT] resultado para [SEARCH_TERM]",
+  "pagefind.alt_search": "Sem resultados para [SEARCH_TERM]. Mostrando resultados para [DIFFERENT_TERM] no lugar",
+  "pagefind.search_suggestion": "Sem resultados para [SEARCH_TERM]. Tente uma das pesquisas a seguir:",
+  "pagefind.searching": "Procurando por [SEARCH_TERM]..."
```

Comment:
Do we actually need to set these here? Pagefind should provide them itself: https://github.com/CloudCannon/pagefind/blob/main/pagefind_ui/translations/pt.json

The aim of the `pagefind.` keys here is mainly for languages Pagefind doesn’t currently support, but it looks like Portuguese translations should be provided.